### ISSUE TITLE
Set SplitView Constraints by absolute number

### ISSUE THREAD

**Issue by richardasmall** [stevengharris/SplitView#10]
Currently it is only possible to set constraints based on fractional sizes of the overall view. Being able to set an absolute size would allow for minimum that can not shrink beyond a certain size requirement.

**Comment by stevengharris** [comment]
I agree this could be useful in some cases. At least in my case, though, it ends up being problematic because there are just too many device variations to account for, and fractions work reasonably well for them. It's probably pretty complicated to sort out, but if you come up with something and/or I can help, let me know.
